test/java/seedu/addressbook/util/TestUtil.java

```diff
@@ -130,4 +130,19 @@ public static void assertTextFilesEqual(Path path1, Path path2) throws IOExcepti
     public static void assertFileDoesNotExist(String filePath) {
         assertTrue(Files.notExists(Paths.get(filePath)));
     }
+
+    @Test
+    public void testIsAnyNull() {
+        //the expected output should be true
+        assertTrue(Utils.isAnyNull((Object)null));
+        assertTrue(Utils.isAnyNull("abc", null));
+        assertTrue(Utils.isAnyNull(1, "abc", null));
+        assertTrue(Utils.isAnyNull(new Object(), null, 1, null));
+
+        //the expected output should be false
+        assertFalse(Utils.isAnyNull(""));
+        assertFalse(Utils.isAnyNull(1, 2));
+        assertFalse(Utils.isAnyNull("abc", 1, 'a'));
+        assertFalse(Utils.isAnyNull(new Object()));
+    }
```

Comment:
In general, a PR should not contain unmerged code from other PRs. Try to limit each PR to one enhancement, which means the PR should be based on a branch that starts off from the `master` branch that is in sync with the upstream `master` branch